### PR TITLE
Remove unnecessary CWL files [issue 119]

### PR DESCRIPTION
**Pull request by GruA**
Files that were removed:
samtools_mpileup.cwl 
position_sort.cwl 
filter_vcf_exac.cwl 
fastq_to_bqsr.cwl 
fastq_align_and_tag.cwl 
deeptools_bamcoverage.cwl 
bam_to_bqsr_no_dup_marking.cwl 
bam_to_bqsr.cwl

### CODE REVIEW

**Review by chrisamiller — APPROVED** [review]
+1   Thanks for your first PR here! 

Closes https://github.com/genome/analysis-workflows/issues/119

**Comment by tmooney** [comment]
Looks like there's a bit more to do with this one.  The tests failed because one of these was used by a file that is still in the repo.

Excerpt from the travis run:
```
Tool definition failed validation:
definitions/subworkflows/downsampled_alignment.cwl:32:1: checking field `steps`
definitions/subworkflows/downsampled_alignment.cwl:43:5:   checking object `definitions/subworkflows/downsampled_alignment.cwl#align_workflow`
definitions/subworkflows/downsampled_alignment.cwl:44:9:     Field `run` contains undefined reference to `file:///home/travis/build/genome/analysis-workflows/definitions/subworkflows/bam_to_bqsr.cwl`
```

File: definitions/pipelines/rnaseq.cwl [hunk 1]
```diff
@@ -134,7 +134,8 @@ steps:
     position_sort:
         run: ../tools/samtools_sort.cwl
         in:
-            bam: merge/merged_bam
+            output_filename: position_sorted_bam
+            input_bam: merge/merged_bam
         out:
             [position_sorted_bam]
```
Comment (chrisamiller) [inline definitions/pipelines/rnaseq.cwl:140]: change this line to `[sorted_bam]`, which is what tools/samtools_sort.cwl returns, and I think you'll be in business.

**Review by chrisamiller — APPROVED** [review]
+1

**Pull request merged**